run-build.ps1

```diff
@@ -24,9 +24,6 @@ if ($PgoInstrument) {
 
 if ($Sign) {
   $Parameters = "$Parameters -sign /p:SignCoreSdk=true"
-
-  # Workaround https://github.com/dotnet/arcade/issues/1776
-  $WarnAsError = $false
```

Comment:
I wasn't exactly sure what was going on here as this is linked to a closed issue from 2019. I haven't seen any detrimental effects from removing this.

Reply:
Seems specific to arcade issue linked. Did you try running a fully signed build against the internal pipeline. Seems like this is the only time you'd hit the problem this was trying to solve